chia/full_node/full_node.py

```diff
@@ -96,7 +100,9 @@ def __init__(
         self.full_node_peers = None
         self.sync_store = None
         self.signage_point_times = [time.time() for _ in range(self.constants.NUM_SPS_SUB_SLOT)]
-
+        self.pending_tx_request = {}
```

Comment:
Can we put this stuff in the full node store instead of full node? I don't want to add a lot of state in this class

Reply:
sure